tools/topology/topology1/sof-rn-rt5682-rt1019.m4

```diff
@@ -21,16 +21,16 @@ include(`platform/amd/acp.m4')
 # Playback pipeline 1 on PCM 0 using max 2 channels of s16le.
 # Schedule 96 frames per 2000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(
-ifdef(`DTS', sof/pipe-dts-codec-playback.m4, sof/pipe-passthrough-playback.m4),
+ifdef(`DTS', sof/pipe-eq-iir-dts-codec-playback.m4, sof/pipe-passthrough-playback.m4),
 	1, 0, 2, s16le,
-	2000, 0, 0,
+	1000, 0, 0,
 	48000, 48000, 48000)
 
 # playback DAI is ACPSP using 2 periods
 DAI_ADD(sof/pipe-dai-playback.m4,
 	1, ACPSP, 0, acp-headset-codec,
 	PIPELINE_SOURCE_1, 2, s16le,
-	2000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
```

Comment:
I wonder whether the period of headset mic pipeline (PIPELINE#2) requires to change accordingly (line45 and line52), since they are the duplex which share the same PCM and DAI_BE.

I'm not sure about this honestly. @balakishorepati should they have the same period?

Reply:
@joechengxperi 
time period for headset playback and capture should be 1.msec.

Reply:
> I wonder whether the period of headset mic pipeline (PIPELINE#2) requires to change accordingly (line45 and line52), since they are the duplex which share the same PCM and DAI_BE.
> 
> I'm not sure about this honestly. @balakishorepati should they have the same period?
There is no hard rule, but better to have same period for both playback and capture.

Reply:
@johnylin76 @bhiregoudar @balakishorepati I didn't commit the 1ms on capture pipeline since I still encounter IPC timed out error when the period is 1ms on capture pipeline and Google assistant is on. I'll add details on issue tracker.

Reply:
@johnylin76 @bhiregoudar @balakishorepati 
I"ve already updated the period to 1ms for both playback and capture pipeline(dmic and headset). Let me know if it's okay to be merged now. Thanks.